.travis.yml

```diff
@@ -6,12 +6,10 @@ cache: bundler
 language: ruby
 rvm:
 - ruby-head
-- 2.2
-- 2.1
-- 2.0.0
```

Comment:
Tests might pass, but is dropping these rubies the right thing? I'm fine with either way, but sneakers has less viability when it only cares to support the latest stable.